docs/faq.md

```diff
@@ -120,6 +120,12 @@ tart ip --resolver=arp <VM>
 
 This causes the `tart ip` to consult the host's ARP table instead of the `/var/db/dhcpd_leases` file.
 
+Note that this method of resolving the IP heavily relies on the level of VM's talkativeness on the network. This is normally not an issue for macOS VMs, but on Linux VMs you might need to install Samba, which includes a [NetBIOS name server](https://www.samba.org/samba/docs/current/man-html/nmbd.8.html) that exhibits the same behavior as macOS and populates the ARP table of the host OS:
```

Comment:
`talkativeness` sounds wrong to me, maybe it is better to describe the actual behavior (sending the special arp request that makes the host added to arp cache).

Reply:
Improved in https://github.com/cirruslabs/tart/pull/1009/commits/2e52d9547fc1d7e65770291499b27dd93e57ab59.